Content.Server/Weapons/Melee/MeleeWeaponSystem.cs

```diff
@@ -57,45 +58,59 @@ public override void Initialize()
         base.Initialize();
         SubscribeLocalEvent<MeleeChemicalInjectorComponent, MeleeHitEvent>(OnChemicalInjectorHit);
         SubscribeLocalEvent<MeleeWeaponComponent, GetVerbsEvent<ExamineVerb>>(OnMeleeExaminableVerb);
+        SubscribeLocalEvent<MeleeWeaponComponent, ExamineGroupEvent>(OnExamineGroup);
     }
 
+    private void OnExamineGroup(EntityUid uid, MeleeWeaponComponent component, ExamineGroupEvent args)
+    {
+        if (component.ExamineGroup != args.ExamineGroup)
+            return;
+
+        if (component.HideFromExamine)
+            return;
+
+        var damageSpec = GetDamage(uid, component);
+
+        if (damageSpec == null)
+            return;
+
+        foreach (var damage in damageSpec.DamageDict)
+        {
+            if (damage.Value != FixedPoint2.Zero)
+            {
+                args.Entries.Add(new ExamineEntry(component.ExaminePriority, Loc.GetString("damage-value", ("type", damage.Key), ("amount", damage.Value))));
+            }
+        }
+    }
     private void OnMeleeExaminableVerb(EntityUid uid, MeleeWeaponComponent component, GetVerbsEvent<ExamineVerb> args)
     {
         if (!args.CanInteract || !args.CanAccess || component.HideFromExamine)
             return;
 
+        if (GetDamage(uid, component) == null)
+            return;
+
+        _examine.AddExamineGroupVerb(component.ExamineGroup, args);
+    }
+
+    private DamageSpecifier? GetDamage(EntityUid uid, MeleeWeaponComponent component)
+    {
         var getDamage = new ItemMeleeDamageEvent(component.Damage);
+
         RaiseLocalEvent(uid, getDamage);
 
-        var damageSpec = GetDamage(component);
+        var damageSpec = component.Damage.Total > FixedPoint2.Zero ? component.Damage : null;
```

Comment:
Removed GetDamage?

Reply:
I had the thought that it wasn't needed since it was only called by one place in the code (and was only one line), but put it back since it does make the code easier to read.